### PR TITLE
Latex Rendering Updates

### PR DESCRIPTION
**Pull request by cwarnermm**
#### Summary
Clarified Latex Rendering config setting descriptions and fixed a broken product documentation link.

#### Ticket Link
https://community-daily.mattermost.com/boards/workspace/g4k9s1f953n4bytz5f9sz9orer/e7651b0e-83ad-4910-874d-875706ddf121/925d0849-da95-4be0-8dd5-f0bf1914e0c3/7assur6sf6h9eessmm55cbwox3a

#### Release Note
```release-note
Clarified Latex Rendering config setting descriptions and fixed a broken product documentation link.
```

### CODE REVIEW

**Review by josephbaylon — APPROVED** [review]
LGTM. Verified help text are updated in admin console.

**Review by michelengelen — APPROVED** [review]
LGTM!

nit: although I am wondering if we could do better with this `... Latex code in code blocks ...`

**Comment by cwarnermm** [comment]
@michelengelen - I'm open to suggestions on how best to differentiate the two latex rendering options now available in-product.

**Comment by michelengelen** [comment]
@cwarnermm thats not what i meant ... it just reads a bit awkward to have two times "code" quickly after one another :D

**Comment by cwarnermm** [comment]
@michelengelen - It's a fair point! @esethna - Thoughts?

**Comment by esethna** [comment]
@cwarnermm we can probably drop the first "code"?

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -2544,7 +2544,7 @@ const AdminDefinition = {
                         label: t('admin.customization.enableLatexTitle'),
                         label_default: 'Enable Latex Rendering:',
                         help_text: t('admin.customization.enableLatexDesc'),
-                        help_text_default: 'Enable rendering of Latex code. If false, Latex code will be highlighted only.',
+                        help_text_default: 'Enable rendering of Latex code in code blocks. If false, Latex code will be highlighted only.',
```
Comment (cwarnermm) [inline components/admin_console/admin_definition.jsx:2547]:
```suggestion
                        help_text_default: 'Enable rendering of Latex in code blocks. If false, Latex code will be highlighted only.',
```

File: i18n/en.json [hunk 2]
```diff
@@ -606,9 +606,9 @@
...
   "admin.customization.enableGifPickerDesc": "Allow users to select GIFs from the emoji picker via a Gfycat integration.",
   "admin.customization.enableGifPickerTitle": "Enable GIF Picker:",
-  "admin.customization.enableInlineLatexDesc": "When true, users may render inline Latex code with dollar signs surrounding the text. When false, Latex can only be rendered in a code block using syntax highlighting. [Learn more about text formatting in our documentation](!https://docs.mattermost.com/messaging/formatting-text.html).",
+  "admin.customization.enableInlineLatexDesc": "Enable rendering of inline Latex code. If false, Latex can only be rendered in a code block using syntax highlighting. Please review our [documentation](!https://docs.mattermost.com/messaging/formatting-text.html) for details about text formatting.",
   "admin.customization.enableInlineLatexTitle": "Enable Inline Latex Rendering:",
-  "admin.customization.enableLatexDesc": "Enable rendering of Latex code. If false, Latex code will be highlighted only.",
+  "admin.customization.enableLatexDesc": "Enable rendering of Latex code in code blocks. If false, Latex code will be highlighted only.",
```
Comment (cwarnermm) [inline i18n/en.json:611]:
```suggestion
  "admin.customization.enableLatexDesc": "Enable rendering of Latex in code blocks. If false, Latex code will be highlighted only.",
```

**Comment by cwarnermm** [comment]
@esethna - Great suggestion! I've updated the text accordingly. Thanks!

**Review by esethna — APPROVED** [review]

**Review by michelengelen — APPROVED** [review]
reads out a lot better ... thanks @cwarnermm ✨

**Pull request merged**